### PR TITLE
[DFFA-4] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by O-mobola**
HTML Forms 
Add page navigations

Screenshot_20210426-130406

Screenshot_20210426-130316

### CODE REVIEW

File: submissions/saudatSalawudeen/html-forms/contact-form.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+				<div class="form-box">
+					 <button type="submit" id="btn">Submit</button>
+				</div>
+			</div>
+		</form>
+	</main>
+	<hr/>
+	
+	<footer>
+		<p><samll>All right reserved &copy; 2021. Zaris&trade;.</samll></p>
```
Comment (Ireti203) [inline submissions/saudatSalawudeen/html-forms/contact-form.html:69]: What is the samll tag for?

**Review by Ireti203 — CHANGES_REQUESTED** [review]
The product.html file should be in the html folder while the forms should be in the html-forms folder

File: submissions/saudatSalawudeen/html-forms/contact-form.html (hunk 1 above)
Comment (O-mobola) [reply submissions/saudatSalawudeen/html-forms/contact-form.html]: @Ireti203, please I have updated the file, thanks.

**Comment by O-mobola** [comment]
I have put the files in their appropriate folders as reviewed.

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**